docs/examples/rx/01_simple_connect/index.js

```diff
@@ -0,0 +1,40 @@
+// Required imports
+const { combineLatest } = require('rxjs');
+const { switchMap } = require('rxjs/operators');
+const { ApiRx } = require('@polkadot/api');
+const { WsProvider } = require('@polkadot/rpc-provider');
+
+// Initialise the provider to connect to the local node
+const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+
+async function main () {
```

Comment:
Doesn't need to be async.

Reply:
if i get rid of `async` it causes error:
```
/Users/me/code/src/polkadot-js/api/docs/examples/rx/01_simple_connect/index.js:40
main().catch(console.error);
      ^

TypeError: Cannot read property 'catch' of undefined
```

Reply:
Only `main()` then at the end? imo if there are errors, they should be handled rxjs-ly. cc @jacogr

Reply:
if I change it from `async function main () {` to `function main () {`, and change `await ApiRx.create()` to `ApiRx.create()`, and change `main().catch(console.error)` to `main()` they work